### DISCUSSION TITLE
Great little utility

### DISCUSSION THREAD

**Discussion by JackMBurch** [w4po/ExplorerTabUtility#8] [General]
This is awesome, thanks for this. Should definitely be something that's native in windows but this is a very helpful workaround. Props!

**Comment by w4po** [comment]
Thank you so much for your encouraging words! I'm glad to hear that you find the workaround helpful.
It's true – native support in Windows would be fantastic.

Your support means a lot. Thanks again! ♥️

**Comment by crispearson** [comment]
Agreed. Thank you so much for creating this!

**Reply by w4po** [reply]
Hi @crispearson,
Thank you for the kind words! Happy to hear you're finding it useful ♥️

**Reply by trACEroam** [reply]
does this work on windows10?

**Comment by Samer-ok** [comment]
So glad I finally found this! I've been searching for a way to accomplish this since they introduced  tabs in windows explorer. I've lost hope that someone actually made this, so glad I finally came across it today. Please keep it going. 
had to make a github account for the first time to say thanks

**Reply by w4po** [reply]
Hi @Samer-ok,

Thank you so much for creating a GitHub account just to share this! Glad it's helping you.
Feel free to suggest any improvements ♥️

**Reply by Samer-ok** [reply]
love the new update, looks more modern and feels smoother, and the regular installer option is much appreciated.
in the previous versions had an issue with recycle bin sometimes not opening if I had other tabs open, but it seems solved in this latest version.  Thanks for the great work.

**Comment by SanjithDev** [comment]
Open source continues to be incredibly powerful, playing a major role in the remarkable growth of the IT field. Your utility has been excellent so far!
